src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.InitializationOptions.cs

```diff
@@ -29,6 +30,7 @@ internal readonly record struct InitializationOptions()
             internal string? Locale { get; init; } = null;
             internal IEnumerable<DiagnosticAnalyzer>? AdditionalAnalyzers { get; init; } = null;
             internal IJsonRpcMessageFormatter? ClientMessageFormatter { get; init; } = null;
+            internal ParseOptions? ParseOptions { get; init; } = null;
```

Comment:
was this actually needed?

Reply:
Yes, partial constructors need Preview lang version to parse correctly.